php/php.editor/src/org/netbeans/modules/php/editor/parser/astnodes/FieldsDeclaration.java

```diff
@@ -78,6 +78,54 @@ public static FieldsDeclaration create(FieldsDeclaration declaration, List<Attri
         );
     }
 
+    /**
+     * Convert from FormalParameter to FieldsDeclaration.
+     *
+     * @param parameter
+     * @return the FieldsDeclaration, can be {@code null} if the parameter
+     * doesn't have a visibility modifier.
+     */
+    @CheckForNull
+    public static FieldsDeclaration create(FormalParameter parameter) {
```

Comment:
Convert from FormalParameter to FiledsDeclaration if needed.
e.g. Scan a converted one in a visitor.